openapi/openapi.yaml

```diff
@@ -1,115 +1,83 @@
-openapi: 3.0.3
```

Comment:
Updated with the output from `/api-yaml`. I can keep doing this or not. Whatevs.